yellowbrick/classifier/learning_curve.py

```diff
@@ -0,0 +1,240 @@
+# yellowbrick.classifier.learning_curve
+# Implements a learning curve visualization for classification
+#
+# Author:   Jason Keung <jason.s.keung@gmail.com>
+# Created:  Mon May 22 09:22:00 2017 -0500
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: learning_curve.py [] jason.s.keung@gmail.com $
+
+"""
+Implementations a learning curve visualizer for classification. 
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.model_selection import learning_curve
+from sklearn.model_selection import ShuffleSplit
+
+from yellowbrick.base import ModelVisualizer
+
+## Packages for export
+__all__ = [
+    "LearningCurveVisualizer", 
+]
+
+##########################################################################
+## LearningCurveVisualizer Visualizer
+##########################################################################
+
+class LearningCurveVisualizer(ModelVisualizer):
+    """
+    Generate a simple plot of the test and training learning curve. 
+    Learning curves demonstrate is a plot of proxy measures for implied
+    learning with experience. 
+
+    The X axis represents experience, or the number of training samples. 
+    The Y axis represents learning, or the train and cross validation scores.
+
+    Parameters
+    ----------
+
+    model : a Scikit-Learn estimator
+    
+    train_sizes: ndarray or Series, default: np.linspace(.1, 1.0, 5)
+        An array that represents the proportion of data for the learning curve
+
+    cv : int, cross-validation generator or an iterable, optional
```

Comment:
This will probably not render correctly. You can use http://rst.ninjs.org/ to test the rendering of your docstrings.